requirements.txt

```diff
@@ -1 +1,11 @@
 -e .
+streamlit
```

Comment:
These do not go here. The `-e .` delegates to the dependencies listed in `setup.py`

Reply:
I added MANIFEST.in to solve the problem.
I confirmed it works in Docker.

Reply:
And I removed package names from requirements.txt